tests/v2/validation/nodescaling/scaling_node_driver_test.go

```diff
@@ -97,16 +110,16 @@ func (s *NodeScalingTestSuite) TestScalingNodePools() {
 	}
 }
 
-func (s *NodeScalingTestSuite) TestScalingNodePoolsDynamicInput() {
-	if s.scalingConfig.MachinePools == nil {
-		s.T().Skip()
-	}
+// func (s *NodeScalingTestSuite) TestScalingNodePoolsDynamicInput() {
+// 	if s.scalingConfig.MachinePools == nil {
+// 		s.T().Skip()
+// 	}
```

Comment:
assuming this can be removed instead of commented out

Reply:
That was an accident I will uncomment it out.